hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServicesSchedulerActivities.java

```diff
@@ -399,7 +442,7 @@ public void testReserveNewContainer() throws Exception {
       JSONObject allocations = getFirstSubNodeFromJson(json,
           FN_SCHEDULER_ACT_ROOT, FN_ACT_ALLOCATIONS);
       verifyQueueOrder(allocations,
-          "root-root.c-root.a-root.b-root.b.b3-root.b.b1");
+          "root-root.a-root.c-root.b-root.b.b3-root.b.b1");
```

Comment:
Why did this change?

Reply:
i modified the order of the config properties